pgml-dashboard/src/components/inputs/text/search/search/search.scss

```diff
@@ -0,0 +1,7 @@
+div[data-controller="inputs-text-search"] {
+    .dropdown {
+        .dropdown-menu {
+            padding: 0;
```

Comment:
Because this re-uses the `Dropdown` component which doesn't support (and shouldn't in my opinion) adding custom classes to an element. This overrides the `Dropdown` style.